index.js

```diff
@@ -40,7 +40,8 @@ function DB (opts) {
     map: function (row, next) {
       if (!row.value) return null
       var v = row.value.v, d = row.value.d
-      if (v && v.lat !== undefined && v.lon !== undefined) {
+      var k = row.value.k
+      if (k && v.lat !== undefined && v.lon !== undefined) {
```

Comment:
I don't quite understand this change, can you explain the implications / reasons?